x-pack/plugins/security_solution/public/resolver/view/map.tsx

```diff
@@ -66,6 +68,10 @@ export const ResolverMap = React.memo(function ({
   const hasError = useSelector(selectors.hasError);
   const activeDescendantId = useSelector(selectors.ariaActiveDescendant);
   const { colorMap } = useResolverTheme();
+  const { cleanUpQueryParams } = useResolverQueryParams();
+  useEffectOnce(() => {
```

Comment:
:grey_question: How does this happen only when it unmounts? Is that part of `useEffectOnce`?

Reply:
no, it's part of useEffect. https://reactjs.org/docs/hooks-effect.html#effects-with-cleanup

Reply:
thats just part of effects. useEffectOnce is just useEffect with `[]` passed as the deps array.